opensquirrel/passes/exporter/cqasmv1_exporter.py

```diff
@@ -14,12 +15,11 @@ class _CQASMv1Creator(IRVisitor):
     # Precision used when writing out a float number
     FLOAT_PRECISION = 8
 
-    def __init__(self, register_manager: RegisterManager) -> None:
+    def __init__(self, register_manager: RegisterManager, barrier_links: list[list[Statement]]) -> None:
```

Comment:
This looks like a code smell. We add a new functionality, and we have to upate the exporter's `__init__` function to cope with it?

Reply:
"code smell", I learned a valuable new term today :)